queries.sql

```diff
@@ -1,3 +1,33 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * 
+FROM animals 
+WHERE name LIKE '%mon%';
```

Comment:
**Changes Required**
- [ ] This query will return any record that contains `mon` yet the requirement dictates that we return only records ending with `mon`. Let us try and correct this as stated in the requirement.